.travis.yml

```diff
@@ -1,16 +1,31 @@
 language: php
 php:
+  - 7.1
   - 7.2
   - 7.3
   - 7.4
 
+env:
+  global:
+    - RUN_PHPSTAN="FALSE"
+  matrix:
+    - PREFER_LOWEST="" WITH_COVERAGE="--coverage-clover=coverage.xml"
+    - PREFER_LOWEST="--prefer-lowest" $WITH_COVERAGE=""
+
 matrix:
+  include:
+    - name: 'PHPStan'
+      php: 7.2
```

Comment:
Why 7.2?

Reply:
copied from `sabre-io/http` - it picked `7.2`

Reply:
I see.

Reply:
Since the phpstan build is rather slow compared to other builds and phpstan bumps its min version regularily I would use at least 7.3 or maybe even 7.4 for the phpstan build.

Reply:
used 7.4 - may as well be bleeding-edge